l1-contracts/test/portals/TokenPortal.sol

```diff
@@ -42,7 +43,7 @@ contract TokenPortal {
 
     // Hash the message content to be reconstructed in the receiving contract
     bytes32 contentHash =
-      Hash.sha256ToField(abi.encodeWithSignature("mint(uint256,bytes32)", _amount, _to));
+      Hash.sha256ToField(abi.encodeWithSignature("mint(uint256,bytes32)", _amount, _to, msg.sender));
```

Comment:
I think using a `canceller` is a nicer solution that `msg.sender`, mainly to cover the cases where calls might happen from contracts that hold aggregated funds, but want to deposit "on behalf" of someone specific.

Agree with @Maddiaa0 that the selectors should be updated for clarity.